src/NLog/Common/ConversionHelpers.cs

```diff
@@ -42,7 +42,7 @@ namespace NLog.Common
     public static class ConversionHelpers
     {
         /// <summary>
-        /// Converts input string value into <see cref="System.Enum"/>
+        /// Converts input string value into <see cref="System.Enum"/>. Parsing is case-insensitive.
```

Comment:
This is wrong now?

Reply:
Not sure I understand. Thought you wanted me to update documentation to match actual behavior.

Reply:
I'm confused I guess

>  I have decided to completely remove the internal TryParse-method without ignoreCase-parameter

So there is a still a tryparse without ignoreCase?

(and if we have the ignoreCase parameter, "Parsing is case-insensitive." is wrong)

PS: the diff made it here conflusing

Reply:
Nope. Your existing public method `TryParseEnum` was case-insensitive already (see the source code). When I implemented parsing of DbType-enum I used an internal method TryParse that was case-sensitive. I have now removed this internal method TryParse, and now using public method `TryParseEnum` (Just like I wrote about putting on my thinking hat).

Reply:
Ok thank for the clarification!